app/api/resources/templates/shared/appbar.html

```diff
@@ -2,11 +2,11 @@
     <div class="topbar-left">
         <div class="home-link">
             <a href="https://beta.ensembl.org/">
```

Comment:
Now this link is where `aria-label` is appropriate, because it doesn't have any text content. Could you add `aria-label="Ensembl home page"` to it?